Source/Core/DolphinQt/HotkeyScheduler.cpp

```diff
@@ -138,6 +140,8 @@ static void HandleFrameStepHotkeys()
     frame_step_count = 0;
     frame_step_hold = false;
     frame_step_delay_count = 0;
+    Settings::Instance().SetIsFrameStepping(false);
+    Core::CallOnStateChangedCallbacks(Core::GetState(Core::System::GetInstance()));
```

Comment:
Directly calling Core::CallOnStateChangedCallbacks is slightly strange... But the alternative would be significantly more code to write, and it's not like doing it this way is all that bad. So treat this comment just as something I wanted to make a note about rather than something you have to change.

Reply:
Actually, I recalled seeing
 ```emit Settings::Instance().EmulationStateChanged(Core::GetState(Core::System::GetInstance()));```
used in another commit.  Since we are only blocking settings signals, this should be fine, right?